src/checkstyle/checkstyle-suppressions.xml

```diff
@@ -42,7 +42,8 @@
 	<suppress files=".*WiremockServerApplicationTests.*" checks="LineLengthCheck"/>
 	<suppress files=".*ContractVerifierConfigProperties.*" checks="JavadocStyle"/>
 	<suppress files=".*YamlContract.*" checks="JavadocVariable"/>
-	<suppress files=".*HttpHeaders.*" checks=".*"/>
-	<suppress files=".*HttpMethods.*" checks=".*"/>
-	<suppress files=".*HttpStatus.*" checks=".*"/>
+	<suppress files=".*HttpHeaders.*" checks="HideUtility*"/>
```

Comment:
Please remove these suppressions. Instead, fix the javadocs and add private constructors to utility classes.

Reply:
If I do that, I'll break backwards compatibility.
See [comment](https://github.com/spring-cloud/spring-cloud-contract/pull/1151#discussion_r313334793) on one of those classes.
I fixed all other checkstyle issues so this the only one that will stay there until the next major release unfortunately.